src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Native_x64_VS2013_Default.dll.sarif

```diff
@@ -734,6 +734,27 @@
               }
             }
           ]
+        },
+        {
+          "ruleId": "BA2027",
+          "ruleIndex": 31,
+          "message": {
+            "id": "Warning",
+            "arguments": [
+              "Native_x64_VS2013_Default.dll",
+              "Library: c:\\Users\\mikefan\\documents\\visual studio 2013\\Projects\\Native_x64_VS2013_Default\\Native_x64_VS2013_Default\\x64\\Debug\\stdafx.obj, Language: Cxx, Version: NotSet\r\nLibrary: c:\\Users\\mikefan\\documents\\visual studio 2013\\Projects\\Native_x64_VS2013_Default\\Native_x64_VS2013_Default\\x64\\Debug\\Native_x64_VS2013_Default.obj, Language: Cxx, Version: NotSet\r\nLibrary: c:\\Users\\mikefan\\documents\\visual studio 2013\\Projects\\Native_x64_VS2013_Default\\Native_x64_VS2013_Default\\x64\\Debug\\dllmain.obj, Language: Cxx, Version: NotSet\r\nLibrary: C:\\Program Files (x86)\\Microsoft Visual Studio 12.0\\VC\\lib\\amd64\\MSVCRTD.lib, Language: Cxx, Version: NotSet\r\n"
```

Comment:
>c:\\Users\\ [](http://example.com/codeflow?start=407&length=11)

this is showing private information.
Let's use just the name of the module/library

Reply:
fixed